.dockerignore

```diff
@@ -1,4 +1,6 @@
 *
+!Cargo.lock
+!Cargo.toml
```

Comment:
Why Cargo.toml too?

Reply:
We need it in the docker image to build the frontend application. Previous all Rust code was inside `web`, now it's being refactored as a Cargo Workspace (aka monorepo, of which `web` is a member) so that rust tools can be run from the root of the repo, and in the future I can split the web app into multiple subpackages for organization and build parallelism.